content/docs/atom-guide/look-dev/materials/wrinkle-masks.md

```diff
@@ -0,0 +1,60 @@
+---
+linktitle: Wrinkle Masks for Skin Materials
+title: "Wrinkle Mask Workflow"
+description: "Learn about how to drive the blending between wrinkle maps with morph targets."
+toc: true
+weight: 200
+---  
+
+This document describes the workflow for driving the blending of different wrinkle layers in the [Skin Material Type](https://github.com/o3de/o3de/blob/development/Gems/Atom/Feature/Common/Assets/Materials/Types/Skin.materialtype) using morph targets. It can be seen in action in the [Old World teaser](https://www.youtube.com/watch?v=ebr8t_mz8p8) that came out with O3DE's launch. Following this workflow will allow for the automatic blending between wrinkle layers driven by the active morph targets of the actor component.
+
+{{< important >}}
+This workflow represents a minimum viable product for animating wrinkle masks. There are a number of aspects that could use feedback on how to improve, such as how the normal maps and blend masks are blended, and the potential need for corrective blend masks. If you are using this feature, please reach out to [sig-graphics-audio on Discord](https://discord.com/channels/805939474655346758/816043793576886273) with your suggestions!
+{{< /important >}}
+
+## Folder Structure and Naming Conventions
+In the folder that `<mycharacter>.fbx` is in, add a `<mycharacter>_wrinklemasks` folder. Using this exact naming convention, the Asset Processor will associate any wrinkle masks in the wrinkle mask folder with morph targets from the .fbx.
+
+![Folder Naming Convention](/images/atom-guide/materials/skin/wrinkle-mask-folder-naming-convention.png)
+
+In the `<mycharacter>_wrinklemasks` folder add one blend mask for each morph target that activates the wrinkle layers. If the morph target does not activate any wrinkle layers at all, it does not need a mask. The name of each mask must match the name of the morph target/blend shape/shape key in the fbx. If you rename the blend shape in the fbx, you must rename the wrinkle mask. Each mask is an independent file, and there is no limitation on how many wrinkle masks you can add to a character, although there will be at most 16 active at any one time.
+
+![Wrinkle Mask File Naming Convention](/images/atom-guide/materials/skin/wrinkle-mask-file-naming-convention.png)
+
+Here you can see that the wrinkle mask file names above match the morph target (shape key in Blender) names below, ending with _wrinklemask.tif.
+
+![Shape Key Names in Blender](/images/atom-guide/materials/skin/shape-key-names-in-blender.png)
```

Comment:
I'd suggest combining the two images together: wrinkle mask file names and morph target names in Blender. And right below, moving and amending the sentence, "In the following image, you can see that the wrinkle mask file names match the morph target names (displayed as shape keys in Blender) and end with `_wrinklemask.tif`."

I'll also check on usage of screenshots of the Blender software and get back to you. Even though you mention Blender in the text, there's concerns that the image can be mistaken for the O3DE interface, and we want to avoid making accidental miscommunications.